libopenage/engine/engine.cpp

```diff
@@ -56,7 +62,7 @@ Engine::Engine(mode mode,
 	// if presenter is used, run it in a separate thread
 	if (this->run_mode == mode::FULL) {
 		this->threads.emplace_back([&, debug_graphics]() {
```

Comment:
If you change the captured arguments to `[&, debug_graphics, wWidth, wHeight]`, you don't need to store width and height as members in `Engine`. You can just pass them through in the function.